src/sentry/static/sentry/app/components/stream/group.tsx

```diff
@@ -420,7 +432,7 @@ const SecondaryCount = styled(({value, ...p}) => <Count {...p} value={value} />)
 const StyledMenuItem = styled(({to, children, ...p}) => (
   <MenuItem noAnchor>
     {to ? (
-      <Link to={to} target="_blank">
+      <Link to={to}>
```

Comment:
nice!..was it incorrect before?

Reply:
said it was not a property? Will take a 2nd look at this

Reply:
`to` means it's a react router link, so it wouldn't make sense to open them in a new tab (unless it was meant to be an external link)

Reply:
I think @taylangocmen intended for this to open in a new tab not sure how to properly do that with typescript tho

Reply:
Use `<ExternalLink />`